### PR TITLE
Include event in subject line

### PR DESCRIPTION
**Pull request by YodaLightsabr**

### CODE REVIEW

File: app/views/donation_mailer/donation_with_message_notification.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <p><%= @emails.length > 1 ? "Hi all," : "Hi there," %></p>
 
 <p>
-  Congrats! We're emailing to let you know that you've received a donation of <%= render_money @donation.amount %> from <%= @donation.name %>. They left you a message:
+  Congrats! We're emailing to let you know that you've received a donation of <a href="<%= "https://hcb.hackclub.com#{@donation.local_hcb_code.url}" %>" target="_blank"><%= render_money @donation.amount %> from <%= @donation.name %></a>. They left you a message:
```
Comment (garyhtou) [inline app/views/donation_mailer/donation_with_message_notification.html.erb]:
```suggestion
  Congrats! We're emailing to let you know that you've received a donation of <a href="<%= url_for @donation.local_hcb_code %>" target="_blank"><%= render_money @donation.amount %> from <%= @donation.name %></a>. They left you a message:
```

give this a shot and see if it works

**Review by sampoder — APPROVED** [review]
hack yeah! lgtm

File: app/views/donation_mailer/donation_with_message_notification.html.erb (hunk 1 above)
Comment (sampoder) [reply app/views/donation_mailer/donation_with_message_notification.html.erb]: Just tested it on my computer seems like it works

File: app/views/donation_mailer/donation_with_message_notification.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <p><%= @emails.length > 1 ? "Hi all," : "Hi there," %></p>
 
 <p>
-  Congrats! We're emailing to let you know that you've received a donation of <%= render_money @donation.amount %> from <%= @donation.name %>. They left you a message:
+  Congrats! We're emailing to let you know that you've received a donation of <a href="<%= url_for @donation.local_hcb_code %>" target="_blank"><%= render_money @donation.amount %> from <%= @donation.name %></a>. They left you a message:
```
Comment (cjdenio) [inline app/views/donation_mailer/donation_with_message_notification.html.erb]:
```suggestion
  Congrats! We're emailing to let you know that you've received a donation of <%= link_to "#{render_money @donation.amount} from #{@donation.name}", @donation.local_hcb_code %>. They left you a message:
```

**Pull request merged**